lib/katello/plugin.rb

```diff
@@ -596,6 +596,12 @@ def katello_template_setting_values(name)
         full_name: N_('Delete Host upon unregister'),
         description: N_("When unregistering a host via subscription-manager, also delete the host record. Managed resources linked to host such as virtual machines and DNS records may also be deleted.")
 
+      setting 'retain_build_profile_upon_unregistration',
+        type: :boolean,
+        default: false,
+        full_name: N_('Retain build profile upon unregistration'),
```

Comment:
The description mentions retaining "media" but looking at the code, I only see `kickstart_repository_id` being retained. Does "media" refer to the kickstart repository serving as the media source, or is there a separate media field that should be handled? Consider clarifying this in the description if "media" is just another term for the kickstart repository.

Reply:
Yeah, Katello does this weird thing where it makes a "fake" media out of the kickstart repository. I think they are the same.